### ISSUE TITLE
myLife: achievements links could be better formatted

### ISSUE THREAD

**Issue by jessewashburn** [open-learning-exchange/planet#7819]
In my opinion, there should not be that space between the link title and the link itself. I would have them both centered

Also, when creating a really long link, it overflows and the link title disappears (It's supposed to say "Really Long Link Title")

![image](https://github.com/user-attachments/assets/24c8009a-a9df-4ca5-ba7d-dd0210213daf)

**Comment by Gavinp14** [comment]
I agree with you, I think the links should be centered as well. I can make this change in the Achievements page revamp.

**Comment by Mutugiii** [comment]
@Gavinp14 kindly confirm if the changes were reflected in the latest myAchievements page revamp
